routes/user.js

```diff
@@ -43,30 +44,34 @@ router.post("/add", (req, res) => {
 router.get("/verify", (req, res) => {
   const email = gUnverifiedEmails.get(req.query.state);
   if (email) {
-    gEmails.add(email);
-    gUnverifiedEmails.delete(req.query.state);
-    res.json({ email, info: `Successfully added ${email}`});
+    Subscribers.addUser(email).then(() => {
+      gUnverifiedEmails.delete(req.query.state);
```

Comment:
Hmm, we should delete the email from gUnverifiedEmails even if addUser fails (force the user to re-do the transaction).

If addUser fails, something is weird with the database and I'd rather not keep user emails around indefinitely in a table whose entries are supposed to be ephemeral.